app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCPublicKeys.scala

```diff
@@ -17,4 +19,33 @@ object DLCPublicKeys {
       fundingPrivKey.publicKey,
       Bech32Address(P2WPKHWitnessSPKV0(payoutKey.publicKey), network))
   }
+
+  def fromExtPubKeyAndIndex(
```

Comment:
So generally speaking, now that we're down to just two keys, I am going to put my vote to get rid of this and move the responsibility of all key derivation concerns to the wallet only. Thoughts? I just think it will make everything cleaner and this really does seem like a wallet-only concern.